Cargo.toml

```diff
@@ -12,11 +12,10 @@ edition = "2018"
 [features]
 now = [ "time" ]
 wasm-bindgen = ["js-sys", "wasm-bindgen_rs", "web-sys"]
+inaccurate = []
 
 [dependencies]
 cfg-if = "1.0"
-
-[target.'cfg(not(any(feature = "stdweb", feature = "wasm-bindgen")))'.dependencies]
```

Comment:
Why do we need to remove this?

Reply:
Sorry, maybe should've done this in a separate PR.
https://doc.rust-lang.org/cargo/reference/config.html#target
> Do not try to match on `debug_assertions` or Cargo features like `feature="foo"`.

Basically, this doesn't do anything.

Reply:
I see, thank you for the info. Yes, let's keep this as a separate PR.

Reply:
Applied requested changes.